src/cljs/rems/spa.cljs

```diff
@@ -217,7 +217,9 @@
    [:div.container
     [:div.navbar
      [:div.navbar-text (text :t/footer)]
-     (when-let [{:keys [version revision repo-url]} (read-current-version)]
+     (when-let [{:keys [version revision repo-url]} {:version "before-removing-conversion-98-gd61fac96"
+                                                     :revision "joo-o"
+                                                     :repo-url "http://www.repotronics.com/"}]
```

Comment:
you forgot to throw away these changes (the "wip" commit)

Reply:
hm, true, done :D